### ISSUE TITLE
react-scripts is not recognized after `npm install`

### ISSUE THREAD

**Issue by batjko** [learnapollo/learnapollo#44]
Hi,

I'm walking through the tutorial for React (great work, btw!) and I find in the later chapters (certainly exercise 5 onwards, not sure if that was the case before that), `npm install` never installs react-scripts, resulting in `npm start` throwing the error, of course.

I always have to manually `npm i -S react-scripts`, after which it then works fine.

Not sure why the initial `npm install` fails to do the job here...

I'm on Win10 64bit, btw.

Cheers.

**Comment by marktani** [comment]
Does that also occur with yarn? What are your versions? Didn't see anything suspicious in the package.json

**Comment by batjko** [comment]
I used yarn for the first one or two exercises, but then ran into the EACCESS issue with read-only folders, already reported in a different issue. So I then switched to npm, but now encounter this.

I am using node v6.9.4, npm 3.10.10 and the repo version is pokedex-react v0.0.1. Yarn is on v0.19.1.

**Comment by marktani** [comment]
> but then ran into the EACCESS issue with read-only folders, already reported in a different issue.

Can't seem to find the issue in question, what are you referring to? 

In which folder are you running the install command?

**Comment by batjko** [comment]
@marktani I'm sorry, the issue I was thinking of was actually on the yarn repo: https://github.com/yarnpkg/yarn/issues/872, which is separate.

I kept getting those access denied errors almost randomly, when using yarn. So I switched back to npm which worked fine, until the react-scripts problem reported here.

**Comment by marktani** [comment]
Hm, I could imagine that the rather old react-scripts version is problematic. 

`react-scripts` is actually a dev dependency and `npm i -S react-scripts` adds a normal depenceny. Can you show a `package.json` of exercise 3 where `npm start` is working?

**Comment by batjko** [comment]
Ok so I tried removing the `node_modules` folders from both exercise-03 and a few later ones, up until exercise06 which is where I stopped.

Then did a new `npm i` and then `npm start` and it worked just fine.

I have a feeling the issue was either a corrupted download during the npm install, or some caching issue perhaps?

Btw, the react-scripts package installed in all folders is 0.4.1.

Anyway, closing the ticket.
